drakrun/drakrun/vm.py

```diff
@@ -91,10 +91,31 @@ def is_running(self) -> bool:
         )
         return res.returncode == 0
 
+    """
+    I have a few suggestions that can help remove the raw xl commands from draksetup
```

Comment:
@chivay @icedevml if you can just tell me if I should go ahead with these then I'll start working on these.